{{cookiecutter.bot_project_name}}/Dockerfile

```diff
@@ -20,11 +30,9 @@ ARG GIT_LOGIN="gitlab-ci-token"
 RUN echo -e "machine ${GIT_HOST}\nlogin ${GIT_LOGIN}\npassword ${GIT_PASSWORD}" > ~/.netrc && \
     apt-get update && \
     apt-get install -y git sudo && \
-    pip install poetry==1.1.6 --no-cache-dir && \
-    poetry config virtualenvs.create false && \
     poetry install --no-dev && \
-    echo "${NON_ROOT_USER} ALL = NOPASSWD: /usr/sbin/update-ca-certificates" > /etc/sudoers.d/express_bot && \
-    rm -rf /root/.cache/pypoetry && \
+    echo "${APP_USER} ALL = NOPASSWD: /usr/sbin/update-ca-certificates" > /etc/sudoers.d/${APP_USER} && \
```

Comment:
There is no need to name a file in `/etc/sudoers.d/` same as username
But maybe name `express_bot` really not very descriptive. I suggest naming it `/etc/sudoers.d/update-ca-certificates` or something like that.

Reply:
Changed